### ISSUE TITLE
Error: The card_present source type with currency eur is not supported in GB.

### ISSUE THREAD

**Issue by Brunowilliang** [stripe/stripe-terminal-react-native#638]
Problem Description:
I'm encountering an issue after connecting to the local_mobile reader in the simulator. When I attempt to execute the payment processing function, I receive the following error:

```shell
 ERROR  Payment Error: [Error: The card_present source type with currency eur is not supported in GB.]
```

Steps to Reproduce:
1. connect to the local_mobile reader using the simulator.
2. execute the function to process the payment.
3. receive the above-mentioned error.

```tsx
const handlePayment = useCallback(async () => {
	const deviceUUID = await DeviceInfo.getUniqueId()
	const deviceBrand = await DeviceInfo.getBrand()
	const deviceModel = await DeviceInfo.getModel()
	const systemVersion = await DeviceInfo.getSystemVersion()
	const deviceType = `${deviceBrand} ${deviceModel} SDK-${systemVersion}`

	setIsProcessing(true)

	try {
		const { paymentIntent, error: createError } = await createPaymentIntent({
			amount: 5 * 100,
			currency: 'eur',
			offlineBehavior: 'prefer_online',
			captureMethod: 'automatic',
			paymentMethodTypes: ['card_present'],
			metadata: {
				name: `Teste Tap - ${page?.id}`,
				UUID: deviceUUID,
				'Device Type': deviceType,
			},
		})

		if (createError) {
			appendError('createError', createError.message)
			throw new Error(createError.message)
		}

		if (!paymentIntent) {
			const message = 'Failed to create payment intent.'
			appendError('paymentIntentError', message)
			throw new Error(message)
		}

		const { error: collectError, paymentIntent: collectPayment } =
			await collectPaymentMethod({
				paymentIntent: paymentIntent,
				enableCustomerCancellation: false,
				skipTipping: true,
			})

		if (collectError) {
			appendError('collectError', collectError.message)
			throw new Error(collectError.message)
		}

		console.log('collectPayment', collectPayment)

		const { error: confirmError, paymentIntent: confirmedPaymentIntent } =
			await confirmPaymentIntent(paymentIntent)

		if (confirmError) {
			appendError('confirmError', confirmError.message)
			throw new Error(confirmError.message)
		}

		if (
			confirmedPaymentIntent &&
			confirmedPaymentIntent.status === 'succeeded'
		) {
			Alert.alert('Success', 'Donation successful!')
		} else {
			const failMessage = 'Payment not succeeded.'
			appendError('paymentStatusError', failMessage)
			throw new Error(failMessage)
		}
	} catch (error) {
		console.error('Payment Error:', error)
		appendError(
			'generalPaymentError',
			error instanceof Error ? error.message : String(error),
		)
	} finally {
		setIsProcessing(false)
	}
}, [
	createPaymentIntent,
	collectPaymentMethod,
	confirmPaymentIntent,
	appendError,
	page?.id,
])
```

Dependencies:
"expo": "~49.0.21",
"@stripe/stripe-terminal-react-native": "^0.0.1-beta.15",

**Comment by nazli-stripe** [comment]
@Brunowilliang this is not an issue with the SDK or integration, but with currency mismatch. In the UK you have to use the currency `gbp` when creating PaymentIntents.

**Comment by nabilfreeman** [comment]
Hey @nazli-stripe - Hope you don't mind me following up on this here.

We have a platform account in GB which uses Stripe Connect Express. We have connected Express accounts in GB, EU and US.

We charge on platform, and then use destination charges to pay out to the connected accounts.

We did get the above error during prototyping Tap to Pay.

Would our Platform account being based in GB limit us to only using Tap To Pay / Terminal for GBP transactions? If so, are there any workarounds or extended permissions we could apply for to enable other currencies?

I can get in touch with our Stripe rep but I thought it would be helpful to write here given the context and public usefulness of the question.

Thanks, Nabs
